bk-register/ROOTFS/opt/sage/beekeeper/register/register.py

```diff
@@ -46,6 +48,15 @@ def register():
     """
     id = flask.request.args.get("id", type=str)
 
+    # check if BEEKEEPER_DB_API is alive
+    try:
+        bk_api_result = requests.get(f'{BEEKEEPER_DB_API}').content
+    except Exception as e:
+        return f"Error: X Beekeeper DB API ({BEEKEEPER_DB_API}) cannot be reached: {str(e)}"
```

Comment:
i dont exactly remember how this works but what happens if you don't include an `error` code in your `return`?  Is the return code "success"?

I was expecting to see something like `return f"Error...", 500`

Reply:
Also I assume you meant to put a number in place of "X" in the string "Error: X Beekeeper DB"

Reply:
Good point, I added the error code! I also replaced X and Y with better error descriptions.